cryptographic_fields/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.5.1'
```

Comment:
Why do we bump the version to `v1.1` instead of `v0.5.2`?

Reply:
I have bumped the version to `v1.1` to be consistent with that repo. Our current version actually is `v.1.0` - https://github.com/enderlabs/django-cryptographic-fields/releases